tests/Directory.Build.props

```
@@ -2,6 +2,10 @@
   <!-- Enable multi-level merging -->
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
 
+  <PropertyGroup>
+    <Nullable>annotations</Nullable>
```

We might want to add a TODO comment here to fix all nullability annotations in the test projects. Otherwise, we can change them to warnings by emptying the `WarningsAsErrors` property instead:
```suggestion
    <WarningsAsErrors></WarningsAsErrors>
```

---

I remember us talking about this when we started the nullable thing, and decided that we wouldn't fix these issues for now, but I think that adding a TODO would be fine 😄

---

Yeah, it's a lot of work to update without any significant benefit... Let's add the comment for now:

```suggestion
    <!-- TODO: Update test projects to use nullable reference types and remove this to enable nullable warnings as errors (the solution default) -->
    <Nullable>annotations</Nullable>
```